pkg/controller/core/resourceflavor_controller.go

```diff
@@ -63,7 +63,7 @@ func (r *ResourceFlavorReconciler) Create(e event.CreateEvent) bool {
 	// As long as one clusterQueue becomes active,
 	// we should inform clusterQueue controller to broadcast the event.
 	if cqNames := r.cache.AddOrUpdateResourceFlavor(flv.DeepCopy()); len(cqNames) > 0 {
-		r.qManager.QueueInadmissibleWorkloads(cqNames)
+		r.qManager.QueueInadmissibleWorkloads(cqNames, true)
```

Comment:
why do we need to force broadcast here?

Reply:
E.g. we have an inactive clusterQueue, workloads submitted to this clusterQueue will stay in heap and won't be popped out. So when we create a flavor, `QueueInadmissibleWorkloads` will always return false and `Broadcast()` won't be called. As a result, if we find an clusterQueue turns active from pending, we should broadcast in any cases.

Reply:
sounds good, lets then just call `r.qManager.Broadcast()` here instead changing the `QueueInadmissibleWorkloads` func and of passing this flag. 

I think we need to invoke `Broadcast` on update as well below.